scaffoldmaker/utils/eftfactory_tricubichermite.py

```diff
@@ -156,24 +156,24 @@ def createEftShellApexBottom(self, nodeScaleFactorOffset0, nodeScaleFactorOffset
         ln_map = [ 1, 1, 2, 3, 4, 4, 5, 6 ]
         remapEftLocalNodes(eft, 6, ln_map)
 
-        assert eft.validate(), 'eftfactory_tricubichermite.createEftShellApexBottom:  Failed to validate eft'
+        assert eft.validate(), 'eftfactory_tricubichermite.createEftShellPoleBottom:  Failed to validate eft'
         return eft
 
-    def createEftShellApexTop(self, nodeScaleFactorOffset0, nodeScaleFactorOffset1):
```

Comment:
As above... very good that you changed the name... please search through other mesh types to ensure they use the new name. I'm sure there are a couple.